### DISCUSSION TITLE
Cannot upload; cannot authenticate

### DISCUSSION THREAD

**Discussion by GJRobert** [gavvvr/obsidian-imgur-plugin#84] [Getting help]
### Cannot upload
1. `<!--⚠️Imgur upload failed, check dev console!-->`
2. Dev console:
```
plugin:obsidian-imgur-plugin:1280 Failed imgur request:  Error: net::ERR_HTTP2_PROTOCOL_ERROR
    at SimpleURLLoaderWrapper.<anonymous> (node:electron/js2c/browser_init:2:49935)
    at SimpleURLLoaderWrapper.emit (node:events:513:28)
uploadFileAndEmbedImgurImage @ plugin:obsidian-imgur-plugin:1280
```

### Cannot authenticate
For trying my luck, I have logged out from the setting panel of Imgur plugin,
and then re-authenticate, but this time on [Authorize a user for OAuth 2.0 - Imgur](https://api.imgur.com/oauth2/authorize?client_id=null&response_type=token):
```
Woah there!
The information you sent wasn't quite correct. Please return to the place that sent you to this page and try again. 
```
<img width="584" alt="截圖 2024-07-26 15 59 15" src="https://github.com/user-attachments/assets/326b2995-8e56-4a5f-8643-029a9c33c547">

### Greetings
Hi there, this is really a handy and essential plugin for Obsidian, and thanks for your contribution.

**Comment by gavvvr** [comment]
Hi @GJRobert. That's because you didn't specify a valid ClientId. First get a Client ID (follow the instructions on the Imgur plugin settings tab), then authenticate. Please let me know if your issue will be resolved.

**Reply by GJRobert** [reply]
Thank you so much for the instruction. I do not recall such a process before or perhaps it was one year ago so I forgot.
Anyway the issue is gone!

**Reply by gavvvr** [reply]
It's a new requirement since recent 2.6.0 update

Having shared Client ID for everyone caused a rate limiting issue for everyone (#80)
